frontend/src/app/containers/App/index.js

```diff
@@ -310,9 +310,13 @@ function sendToGA(type, dataIn, evt = null) {
     }
   };
   if (window.ga && ga.loaded) {
+    let installed = {};
     const chosenTest = getChosenTest();
     data.hitType = type;
     data.hitCallback = hitCallback;
+    data.dimension1 = (window.localStorage.getItem("hasAddon") === "true");
+    data.dimension2 = installed = JSON.parse(window.localStorage.installations);
```

Comment:
I feel like the JSON.parse bit here should be moved up into the let installed line.

Reply:
oh yeah, that was a remnant of some `window` variable checking I was doing but realized wasn't necessary, pushing a fix in 3, 2, 1...